src/Crawler.php

```diff
@@ -52,6 +55,9 @@ class Crawler
     /** @var int|null */
     protected $maximumDepth = null;
 
+    /** @var bool */
+    protected $ignoreRobots = false;
```

Comment:
I think the code will be clearer if we use something positive here. Let's rename it to `respectRobots` and set it to `true`.